gst/nnstreamer/include/tensor_typedef.h

```diff
@@ -257,6 +274,13 @@ typedef struct
   uint32_t dimension[NNS_TENSOR_META_RANK_LIMIT];
   uint32_t format;
   uint32_t media_type;
+
```

Comment:
It would be better if the information about the union could be provided in the comments.

Reply:
I added brief comment on it: `@brief Union of the required information for processing each tensor "format".`
Thanks!